terraform/clickstream_analytics/main.tf

```diff
@@ -0,0 +1,228 @@
+#  Copyright 2024 Google LLC
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+locals {
+  dataflow_service_account = "my-dataflow-sa"
+  max_dataflow_workers     = 1
+  worker_disk_size_gb      = 200
+  machine_type             = "g2-standard-4"
+}
+
+
+// Project
+module "google_cloud_project" {
+  source          = "github.com/GoogleCloudPlatform/cloud-foundation-fabric//modules/project?ref=v32.0.0"
+  billing_account = var.billing_account
+  project_create  = var.project_create
+  name            = var.project_id
+  parent          = var.organization
+  services = [
+    "cloudbuild.googleapis.com",
+    "dataflow.googleapis.com",
+    "monitoring.googleapis.com",
+    "pubsub.googleapis.com",
+    "autoscaling.googleapis.com",
+    "artifactregistry.googleapis.com",
+    "bigtableadmin.googleapis.com",
+    "bigquery.googleapis.com"
+  ]
+}
+
+resource "google_bigtable_instance" "clickstream-analytics" {
+  name = "clickstream-analytics"
+
+  cluster {
+    cluster_id   = "clickstream-analytics-c1"
+    num_nodes    = 1
+    storage_type = "HDD"
+    zone    = "us-central1-c"
```

Comment:
Don't hardcode locations in the TF code, use a variable or local configuration option.